app/code/Magento/Sales/Block/Adminhtml/Order/Create/Form/Address.php

```diff
@@ -388,4 +391,14 @@ public function getAddressAsString(\Magento\Customer\Api\Data\AddressInterface $
 
         return $this->escapeHtml($result);
     }
+
+    /**
+     * Return address store id.
+     *
+     * @return int
+     */
+    protected function getAddressStoreId()
```

Comment:
Unfortunately, we are not allowed to add new public and protected methods to the classes according to our [Backward Compatible Development Guide](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/)

Reply:
This is not an API class.